### DISCUSSION TITLE
QR code help

### DISCUSSION THREAD

**Discussion by laurant94** [nativescript-vue/nativescript-vue#795] [Q&A]
**Hello**. 
I am new to **nativeScript-vue** and would like to know if anyone know how i can implement a QR code reader. 
Thank you

**Comment by rigor789** [comment]
Just use one of the existing plugins, like https://market.nativescript.org/plugins/@nativescript-community/ui-barcodeview/ for example.

**Reply by laurant94** [reply]
Thank you for your reply.
I have already used the package but I don't know how to get the scanner code.
Instead of a code in the @scanResult callback, I get the event.

the event contains an _observer and contains in resultCode which is always empty.

**Reply by rigor789** [reply]
@laurant94 That's probably better to ask in the https://github.com/nativescript-community/ui-barcodeview repository along with the code you have used.

Their example does this: 
```js
onScanResult(evt) {
  console.log(`onScanResult: ${evt.text} (${evt.format})`);
},
```
_https://github.com/nativescript-community/ui-barcodeview/blob/8e150607c61c902757425469216777ac4d458125/demo-vue/app/components/App.vue#L42-L44_

**Reply by laurant94** [reply]
Thank you rogor789.
I received the text
